common/src/main/res/values/strings.xml

```diff
@@ -815,6 +815,7 @@
     <string name="remote_debugging">WebView Remote Debugging</string>
     <string name="remote_debugging_summary">Allow remote debugging of WebView to troubleshoot Home Assistant frontend issues</string>
     <string name="websocket_setting_home_wifi">On Home WiFi Only\n\nNotifications will be delivered by Google only when not on Home WiFi.</string>
+    <string name="websocket_setting_home_wifi_minimal">On Home WiFi Only\n\nNotifications will only be delivered when on Home WiFi.</string>
```

Comment:
For consistency with other descriptions for the minimal version, maybe use "received" instead of "delivered"?